src/App.tsx

```diff
@@ -1,26 +1,46 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from './app/store';
+import { setTodos } from './features/todos';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [listLoading, setListLoading] = useState(false);
+  const dispatch = useDispatch();
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const currentTodo = useAppSelector(state => state.currentTodo);
```

Comment:
Ensure that the `useAppSelector` hook is correctly implemented in the `./app/store` module. It should be a custom hook that selects the `currentTodo` from the Redux state.